string arduinoFeedback
string opcuaFeedback
string errorMessage
